### DISCUSSION TITLE
2021/04/24/restart-from-the-very-beginning-for-electron-with-monitor-build-and-publish/index

### DISCUSSION THREAD

**Discussion by giscus** [sumy7/sumy7.github.io#5] [Announcements]
# 2021/04/24/restart-from-the-very-beginning-for-electron-with-monitor-build-and-publish/index

SumyGG's Blog

https://sumygg.com/2021/04/24/restart-from-the-very-beginning-for-electron-with-monitor-build-and-publish/index.html

**Comment by zhifou** [comment]
不错！

**Comment by zhifou** [comment]
测试一下
